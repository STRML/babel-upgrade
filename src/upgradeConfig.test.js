const upgradeConfig = require('./upgradeConfig');
const babelrcFixture = require('./babelrc');

test('packages', () => {
  expect(upgradeConfig(babelrcFixture)).toMatchSnapshot();
});
