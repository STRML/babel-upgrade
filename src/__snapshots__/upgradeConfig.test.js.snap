// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`packages 1`] = `
Object {
  "plugins": Array [
    "@babel/plugin-transform-function-bind",
    Array [
      "@babel/plugin-transform-object-rest-spread",
      Object {
        "useBuiltIns": true,
      },
    ],
    "@babel/plugin-transform-class-properties",
  ],
  "presets": Array [
    "@babel/preset-es2015",
    Array [
      "@babel/preset-env",
      Object {
        "modules": false,
      },
    ],
    "@babel/preset-react",
  ],
}
`;
